frontend/catalyst/jax_tracer.py

```diff
@@ -327,6 +356,18 @@ def has_nested_tapes(op: Operation) -> bool:
     )
 
 
+def trace_function(ctx, fun, *args, expansion_strategy, **kwargs):
+    """Trace Python function supporting variable dimensions in its arguments and/or results"""
```

Comment:
Maybe add that it should not contain any quantum code, right?

Reply:
Indeed, thanks. Done.